notebooks/ltx-video/ltx-video.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<p>&gt;&gt;In this example we show how to convert text-to-video pipeline in OpenVINO format and run inference.</p><p> </p><p>I see also weight compression in logs, please also mention nncf in this description</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2686/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Adde</p>